src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -75,6 +75,9 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFilterLine(new VCFFilterHeaderLine(VCFConstants.PASSES_FILTERS_v4, "Site contains at least one allele that passes filters"));
 
         addFilterLine(new VCFFilterHeaderLine(NAY_FROM_YNG, "Considered a NAY in the Yay, Nay, Grey table"));
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES, "Site has an excess of alternate alleles"));
```

Comment:
if the excess alleles number (6) was a constant somewhere, you could put that into the header. (maybe users don't care, but i would personally find it helpful!)